core/scss/components/sidebar/sidebar.scss

```diff
@@ -32,7 +32,14 @@
       padding-top: $top-bar__height--desktop;
     }
 
-    &__footer,
+    &__footer {
+      width: inherit;
+      display: flex;
+      flex-direction: column;
+      padding: 0;
+      min-height: rem-calc(130);
```

Comment:
Should footer have min-height? Shouldn't it be based on content?

Reply:
padding bottom added instead 1rem